src/npm-version.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Determines the NPM version, if available
+ *
+ * @return {String} - The full version as a string, or undefined if not available
+ */
+function getVersion() {
+  var version = undefined;
+  if (process.env.npm_config_user_agent && process.env.npm_config_user_agent.match(/.*npm\/.+/)) {
+    var agent = process.env.npm_config_user_agent.split(' ')
+    for (var index in agent) {
```

Comment:
Please do `var… of` rather than `in`, see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#Difference_between_for...of_and_for...in).